index.html

```diff
@@ -19,7 +19,7 @@
     <div class="header">
       <div class="header__left">
         <i class="material-icons"> menu </i>
-        <img src="./assets/yt-logo.png" alt="Logo" />
+        <img src="./images/yt-logo.png" alt="Logo" />
       </div>
       <div class="header__search">
         <form action="">
```

Comment:
In the form tag try to use the method.

Reply:
As requested by the previous reviewer, try to as be clear and polite as possible with your reviews. In this case, explain what do you mean by method, also, please make sure to select the right part of the code to highlight in your comment. This will make your review more direct and explicit, helping the student to understand what and why he should change 😄.

Reply:
Kindly, In the form tag try to use the method.